packages/medusa/src/api-v2/admin/regions/query-config.ts

```diff
@@ -18,6 +20,9 @@ export const defaultAdminRegionFields = [
 ]
 
 export const retrieveTransformQueryConfig = {
+  defaultFields: defaultAdminRegionFields,
+  defaultRelations: defaultAdminRegionRelations,
+  allowedRelations: allowedAdminRegionRelations,
```

Comment:
suggestion(nit): we could just have `defaultAdminRegionFields` which already include the relations. In the route (for now but in the middleware would be better) we can use the util to get the select and relations from the fields. wdyt? Just in order to start the cleanup and not have to maintain two array etc and facilitate the migration later